src/SwaggerUIAssetOverrides.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace light\swagger;
+
+use yii\web\AssetBundle;
+
+class SwaggerUIAssetOverrides extends AssetBundle
+{
+    public $sourcePath = '@vendor/light/yii2-swagger/src';
+
+    public $js = [
+    ];
+
+    public $css = [
+        'swagger-overrides.css',

```

Comment:
The css file is not exists?

Reply:
Sorry, I forgot to remove SwaggerUIAssetOverrides.php